components/textfield/package.json

```
@@ -16,15 +16,16 @@
     "build": "gulp"
   },
   "peerDependencies": {
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^4.0.0"
```

```suggestion
    "@spectrum-css/tokens": "^6.0.0"
```

Good catch @castastrophe - @lunarfusion we should update these both so that they point to the latest release (which is now `6.0.0`)

---

Done